docs/backend-api/backend-api-installation.md

```diff
@@ -0,0 +1,55 @@
+# Backend API Installation
+
+*Please follow these instructions with clean git working tree (you can verify with `git status`). If the installation fails, you can easily revert all changes and start again.*
+
+*If you installed the Shopsys Framework with docker please run all commands in the `php-fpm` docker container*
+
+Require `shopsys/backend-api` package in the `composer.json` or run
+
+```bash
+composer require shopsys/backend-api
+```
+
+Backend API installation is fully automated, so you have to run only (you have to be in the project root directory)
+
+```bash
+./vendor/shopsys/backend-api/install/install.sh
```

Comment:
It would be nice to register the script as [a Composer vendor binary](https://getcomposer.org/doc/articles/vendor-binaries.md) for easier and more standard usage, but that's just a nice-to-have.